openai/api_resources/image.py

```diff
@@ -88,9 +107,9 @@ def _prepare_create_variation(
             api_version=api_version,
             organization=organization,
         )
-        _, api_version = cls._get_api_type_and_version(api_type, api_version)
+        api_type, api_version = cls._get_api_type_and_version(api_type, api_version)
 
-        url = cls._get_url("variations")
+        url = cls._get_url("variations", None, api_type=api_type, api_version=api_version)
```

Comment:
Unless the second parameter is a positional-only parameter (unlikely), I'd indicate the parameter name.

Reply:
fixed